examples/rpc/partner-plugin/services/data/v2/shared_data.go

```diff
@@ -0,0 +1,76 @@
+package helpers
+
+import (
+	"encoding/json"
+	"log"
+	"os"
+	"path/filepath"
+	"sync"
+
+	accommodationv2 "buf.build/gen/go/chain4travel/camino-messenger-protocol/protocolbuffers/go/cmp/services/accommodation/v2"
+)
+
+var (
+	properties []accommodationv2.PropertyExtendedInfo
+	loadOnce   sync.Once
+)
+
+func LoadPropertiesMockData() []accommodationv2.PropertyExtendedInfo {
+	loadOnce.Do(func() {
```

Comment:
if you don't want to use embed, there is also
`func init() {}`
which is called at runtime, when package is first imported

Reply:
Created new package -> mock_data